api/api/src/main/scala/uk/ac/wellcome/platform/api/controllers/WorksController.scala

```diff
@@ -129,13 +130,28 @@ abstract class WorksController[M <: MultipleResultsRequest[W],
     def searchFunction: (Index, WorksSearchOptions) => Future[
       Either[ElasticError, ResultList]] =
       request.query match {
-        case Some(queryString) => worksService.searchWorks(queryString)
-        case None              => worksService.listWorks
+        case Some(queryString) =>
+          worksService.searchWorks(
+            queryTypeToWorkQuery(queryString, request._queryType))
+        case None =>
+          worksService.listWorks
       }
 
     searchFunction(index, worksSearchOptions)
   }
 
+  private def queryTypeToWorkQuery(
+    queryString: String,
+    maybeQueryType: Option[String]): WorkQuery = {
+    maybeQueryType.map(_.toUpperCase) match {
+      case Some("JB") => JustBoostQuery(queryString)
+      case Some("BB") => BroaderBoostQuery(queryString)
+      case Some("SP") => SlopQuery(queryString)
+      case Some("MM") => MinimumMatchQuery(queryString)
+      case _          => SimpleQuery(queryString)
```

Comment:
What’s the thinking behind these query parameters?

Personally I’d lean towards more expressive terms (justboost, broaderboost, slop, minmatch), but I don't feel strongly about it.

Reply:
Yeah, i'm with Alex - just makes it a little clearer to anyone not directly involved.

Reply:
👍